lib/src/auth.dart

```diff
@@ -335,12 +343,17 @@ class WebAuthenticator extends Authenticator {
     assert(_redirect != null);
   }
 
+  WebAuthenticator._restore(DRAWConfigContext config, String credentialsJson)
+      : _redirect = Uri.parse(config.redirectUrl),
+        super.restore(config, new oauth2.Credentials.fromJson(credentialsJson));
+
   static WebAuthenticator create(
-      DRAWConfigContext config, oauth2.AuthorizationCodeGrant grant) {
-    final WebAuthenticator authenticator =
-        new WebAuthenticator._(config, grant);
-    return authenticator;
-  }
+          DRAWConfigContext config, oauth2.AuthorizationCodeGrant grant) =>
+      new WebAuthenticator._(config, grant);
+
+  static WebAuthenticator restore(
```

Comment:
Why did we decide to wrap these public functions wrappers to private implementations?

Reply:
In this case, I'm actually not sure anymore haha.